server/master/src/main/java/org/apache/accumulo/master/replication/RemoveCompleteReplicationRecords.java

```diff
@@ -216,4 +220,13 @@ protected long removeRowIfNecessary(BatchWriter bw, SortedMap<Key,Value> columns
 
     return recordsRemoved;
   }
+
+  /*
+   * Metrics calls snapshot. Largest time stamp will be found for each snapshot then reset.
+   */
+  private void collectLatency(Long createdTime) {
+    long latency = System.currentTimeMillis() - createdTime;
```

Comment:
> I am not sure, but I think this createdTime may come from a persisted source

I believe createdTime comes from the TabletServer, likely calling currentTimeMillis(). As long as we're reporting non-negative latencies, I think it will be fine (e.g. take the max of 0 and the computed latency).

The createdTime is coming from a remote JVM (the tserver), so I dont' think there's a need to use nanoTime here for the monotonically increasing time. We can just fudge it.